_opensearch/rest-api/nodes-apis/nodes-usage.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: Nodes usage
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 40
+---
+
+# Nodes usage
+
+The nodes usage endpoint returns low-level information about REST action usage on nodes.
+
+## Path and HTTP methods
+
+```
+GET _nodes/usage
+GET _nodes/<node id>/usage
+GET _nodes/usage/<metric>
+GET _nodes/<node id>/usage/<metric>
+```
+
+## Path parameters
+
+You can include the following optional path parameters in your request:
+
+Parameter | Type | Description
+:--- | :--- | :---
+node_id | String | A comma-separated list of node ids used to filter results. Supports [node filters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/nodes-apis/index/#node-filters). Defaults to `_all`.
+metric | String | The metrics that will be included in the response. You can set the string to either `_all` or `rest_actions`. `rest_actions` will return the total of the number of times an action has been called on the node. `_all` will return all stats from the node. Defaults to `_all`.
+
+## Query parameters
+
+You can include the following optional query parameters in your request:
+
+Parameter | Type | Description
+:--- | :---| :---
+timeout | Time | Sets the time limit for a response from the node. Default is `30s`.
+master_timeout | Time | Sets the time limit for a response from the cluster manager. Default is `30s`.
```

Comment:
This should be `cluster_manager_timeout` for versions 2.0 or above. `master_timeout` for previous versions. Also, the descriptions should match. So, if the parameter is `cluster_manager_timeout`, the description should say "cluster manager", and if the parameter is `master_timeout`, the description should say "master node".